packages/runtime-core/__tests__/apiWatch.spec.ts

```diff
@@ -69,6 +69,24 @@ describe('api: watch', () => {
     expect(dummy).toMatchObject([1, 0])
   })
 
+  it('watching single source: array', async () => {
+    const array = reactive([] as number[])
+    let dummy
+    watch(array, (array, prevArray) => {
```

Comment:
the test should rather pass a spy `jest.fn()` and check the amount of times the spy is called and with what arguments

Reply:
okay.. let me change it. i just copied from the convention of the previous tests 🙈

Reply:
updated the test